### PR TITLE
[CHG] replace many2one field term by float loan term

### PR DESCRIPTION
**Pull request by houssine78**
replace many2one field term by float loan term

### CODE REVIEW

File: easy_my_coop_loan/models/loan.py [hunk 1]
```diff
@@ -38,6 +28,7 @@ def _compute_subscribed_amount(self):
     user_id = fields.Many2one('res.users',
                               string="Responsible")
     term_date = fields.Date(string="Term date")
+    loan_term = fields.Float(string="term of the loan")
```
Comment (robinkeunen) [inline easy_my_coop_loan/models/loan.py:31]: So the unit of loan_term is _year_? Might be good to put a help message for it.

File: easy_my_coop_loan/models/loan.py [hunk 2]
```diff
@@ -183,7 +172,7 @@ def compute_loan_interest(self):
                 }
             self.get_interest_vals(line, vals)
 
-            rounded_term = int(self.term.term)
-            if self.term.term - rounded_term > 0:
+            rounded_term = int(self.loan_term)
```
Comment (robinkeunen) [inline easy_my_coop_loan/models/loan.py:175]: if you're rounding here and at line 140, maybe you should use an integer instead of a float?

**Review by robinkeunen — CHANGES_REQUESTED** [review]
I'm not sure float is the good type since you're rounding the value when you're using it?

File: easy_my_coop_loan/models/loan.py (hunk 1 above)
Comment (houssine78) [reply easy_my_coop_loan/models/loan.py:31]: For now it is... that why it's a float. It has been asked to pass it in monts but i'm not sure it's a good idea

File: easy_my_coop_loan/models/loan.py (hunk 2 above)
Comment (houssine78) [reply easy_my_coop_loan/models/loan.py:175]: Nope I'm rounding it just to know if the term is of type integer and to handle it like this. In the case of the term is 4,5 years we should take that case too. It's planned but not yet implemented. So for me it should stay of type float for now.

**Review by robinkeunen — APPROVED** [review]

**Pull request merged**